tutorials/scripting/debug/output_panel.rst

```diff
@@ -47,13 +47,17 @@ is controlled by the **Run > Output > Always Clear Output on Play** editor setti
 Additionally, you can manually clear messages by clicking the "cleaning brush" icon
 in the top-right corner of the Output panel.
 
+.. _doc_output_panel_printing_messages:
+
 Printing messages
 -----------------
 
 Several methods are available to print messages:
 
 - :ref:`print() <class_@GlobalScope_method_print>`: Prints a message.
   This method accepts multiple arguments which are concatenated together upon printing.
+  This method has variants that separate arguments with tabs and spaces respectively:
+  :ref:`prints() <class_@GlobalScope_method_prints>` and :ref:`printt() <class_@GlobalScope_method_printt>`.
```

Comment:
Those two should be `printt()` first and `prints()` second to match "tabs and spaces respectively".